lib/jsdom/living/events/Event-impl.js

```diff
@@ -44,7 +45,7 @@ class EventImpl {
   }
 
   set returnValue(v) {
-    if (this.cancelable && v === false) {
+    if (this.cancelable && !this._inPassiveListenerFlag && v === false) {
```

Comment:
The first two are described in [set the canceled flag](https://dom.spec.whatwg.org/#set-the-canceled-flag). Would it be better to extract them to a `_setTheCanceledFlag()` method, perhaps?

The last one I assumed was required based on the other flags already being included there, but it can be removed it if that's not the case.

Reply:
Ah wow, yeah, I totally missed that. Let's extract them, yeah. (Also need to rebase for the shadow DOM changes, I guess.)

Based on https://dom.spec.whatwg.org/#concept-event-initialize not setting the flag, we probably should not. I wonder if it'd be possible to write a test that fails if you add the extra reset, like you did?